### DISCUSSION TITLE
What is the maximum number carousel can preview

### DISCUSSION THREAD

**Discussion by mohamed-elsaeed** [twbs/bootstrap#41043] [Q&A]
I have a business case where my client needs to preview 100 images in the same carousel, can this be achieved using carousel component or should I use a different approach?

**Comment by uncleejay** [comment]
The Bootstrap carousel component can technically handle displaying 100 images, but it's not the most efficient or user-friendly approach, especially for such a large number of images. An alternative approach could be paginated galleries. Break the 100 images into smaller groups and use pagination or tabs to switch between different sets of images. You should avoid using a single large Bootstrap carousel with all images loaded at once.

**Comment by jonhubby** [comment]
Consider implementing pagination or lazy loading to manage the images better. This way, users can navigate through smaller sets of images more easily.
